src/main/java/wanya/Launcher.java

```diff
@@ -0,0 +1,11 @@
+package wanya;
```

Comment:
I like that you have created and neatly organised all your classes into a few packages as it makes it easy for readers to understand the purpose of each of your classes.